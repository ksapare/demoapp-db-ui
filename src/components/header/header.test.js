import Header from './header.component'

import { render, screen } from '@testing-library/react';
import { MemoryRouter } from 'react-router-dom';

describe("Header component", () => {

    it("should render correctly", () => {


        render(
            <MemoryRouter>
                <Header />
            </MemoryRouter>
            )
        expect(screen.getAllByRole('link')).toMatchSnapshot()
    
    })
})