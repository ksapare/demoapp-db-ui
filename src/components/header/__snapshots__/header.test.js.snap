// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component should render correctly 1`] = `
Array [
  <a
    class="option"
    href="/"
  >
    Home
  </a>,
  <a
    class="option"
    href="/get-started"
  >
    Getting Started
  </a>,
]
`;
